.github/workflows/maven.yml

```diff
@@ -1,6 +1,6 @@
 name: Java CI
 
-on: [push, fork]
+on: [push, fork, pull_request]
```

Comment:
I'd initially held off on adding `pull_request` here as it seemed to have quite some activity types that might then also cause new pipelines to be kicked off: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
```
- assigned
- unassigned
- labeled
- unlabeled
- opened
- edited
- closed
- reopened
- synchronize
- converted_to_draft
- ready_for_review
- locked
- unlocked
- review_requested
- review_request_removed
- auto_merge_enabled
- auto_merge_disabled
```
But with the low volume of PRs here we can just see how this goes; better to have some runs than none. Thanks again!

Reply:
It looks like the pull_request is holding the PR from being merged when it doesn't pass. I am not sure why I should include any more of the listed ones.

Reply:
Seems to be holding up well enough ; ready to merge if you ask me.